docs/source/HowTo/DataManagement/metadata.rst

```diff
@@ -63,22 +66,24 @@ used in data selections.
           NewMetaInt : INT
            ATMetaFlt : float
 
-- Finding files with selection by metadata::
+How to find files with selection by metadata
+--------------------------------------------
+
+From the CLI::
 
    find <meta selection>
 
-  For example::
+For example::
 
    FC:/vo.formation.idgrilles.fr/user/a/atsareg> find ATMetaInt=10,11 ATMetaInt1<15
    Query: {'ATMetaInt': {'in': [10, 11]}, 'ATMetaInt1': {'<': 15}}
    /vo.formation.idgrilles.fr/user/a/atsareg/newDir/wms_output.py
 
-8.2 File Provenance Metadata
-----------------------------
 
-In the File Catalog you can declare ancestor files for a given file. This is often needed
-in order to keep track of the derived data provenance path. The ancestor declaration is done
-as following::
+How to declare file's ancestors
+------------------------------
```

Comment:
There are not enough dashes

Reply:
Updated, thanks

Reply:
Shouldn't that create a warning btw ? I did not see any

Reply:
No there was a warning:
```
source/HowTo/DataManagement/metadata.rst:84: WARNING: Title underline too short.
```
Just in this branch there are still too many...